build.gradle.kts

```diff
@@ -1,3 +1,5 @@
+@file:Suppress("HasPlatformType")
```

Comment:
Do we need this and if so, why?

Reply:
It is to suppress these warnings:

![image](https://user-images.githubusercontent.com/5797900/233703829-3d65f0a4-9fde-4092-87be-e0fde9352ea2.png)

Alternatively, we could specify the type explicitly. For example:

```kotlin
val consumableGradleScripts: Configuration by configurations.creating {
  ...
}
```

Reply:
Thanks for the insights.